src/blocks/SliderNew/Slider.tsx

```diff
@@ -124,18 +153,22 @@ export const SliderNewBlock = ({
                 </Swiper>
                 {arrows && !isLocked && (
                     <Fragment>
-                        <Arrow
-                            className={b('arrow', {prev: true})}
-                            type="left"
-                            onClick={onPrev}
-                            size={arrowSize}
-                        />
-                        <Arrow
-                            className={b('arrow', {next: true})}
-                            type="right"
-                            onClick={onNext}
-                            size={arrowSize}
-                        />
+                        <div aria-hidden={withAutoplay}>
+                            <Arrow
+                                className={b('arrow', {prev: true})}
+                                type="left"
+                                onClick={onPrev}
+                                size={arrowSize}
+                                extraProps={{tabIndex: withAutoplay ? -1 : 0}}
```

Comment:
`{tabIndex: withAutoplay ? -1 : 0}`  save in constant  because use in different places